LICENCE.txt

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (C) 2013 Crown Copyright (Government Digital Service)
```

Comment:
Looking at that page, I'd say that this applies:
```
In the case of work which is continually updated, (for example a web site), the year of publication may be shown as a period from first publication until the most recent update, (i.e. 2000-2015)
```

But I don't really mind either way.